Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -869,38 +878,37 @@ ioi ld   (NBCR),a
    inc   hl
    ld    (ix+[_decb]+pd_rx_error),hl
 
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Again, where in the R6000 is this documented?
+#if !defined(_RAB6K)

```

Comment:
I still don't like that it's not documented in the manual.  And maybe I don't understand it, but why would there be a length check error bit and crc error bit in the transmit extra status register (NBTESR)?  I would think those are receiver only things?